pages/HomeScreen.tsx

```diff
@@ -88,7 +91,7 @@ export function HomeScreen({ navigation }) {
           hapticStrength = Haptics.ImpactFeedbackStyle.Light;
           break;
       }
-      hapticStrength && Haptics.impactAsync(hapticStrength);
+      hapticStrength !== false && Haptics.impactAsync(hapticStrength);
```

Comment:
**question:** @filiptronicek Is `hapticStrength && Haptics.impactAsync(hapticStrength)` enough instead of checking its not false

Reply:
I think I'll leave it as is, it may be refactored, but I think there may be some conflicts if I didn't strictly check for `false`.